guide/layouts/default.haml

```diff
@@ -0,0 +1,12 @@
+!!! 5
```

Comment:
what's that ?!

Reply:
doctype html5 which I think is what we want

https://haml.info/docs/yardoc/file.REFERENCE.html#doctype-